puuuub/bin/main.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:http/http.dart' as http;
+import 'package:html/parser.dart';
+
+main() async {
+  final response = await http.Client().get(Uri.parse('https://example.com/'));
+
+  var document = parse(response.body);
+  print(document.getElementsByTagName('title')[0].innerHtml);
```

Comment:
It's better to use querySelector, because there is only one <title> element. Also, since we want to display the text, you can use text instead of innerHtml:

```dart
print(document.querySelector('title).text);
```